### PR TITLE
Fix navigation flow of faq fragment

### LINKED ISSUE #2376: Fix navigation flow of faqs fragment
**Actual Behaviour**

When user selects faqs in drawer layout, if user is not logged in, user is redirected to login activity. If user does not want to login then user need to do two clicks, One for exiting login activity and one to go in other activity.

**Expected Behaviour**

There should be login button in faq fragment. When user is not logged in and if user wants to login, user is redirected to login activity.

**Screenshots of the issue**

**Would you like to work on the issue?**

yes

### PR DESCRIPTION
**Pull request by ParthivMangukiya**
Fixes #2376

Changes: Added Login Button in faqs fragment when user is not logged in. Remove redundant
login_to_see_faqs string in strings.xml.

Screenshots for the change:

### CODE REVIEW

**Comment by iamareebjamal** [comment]
I think we can use a common login to continue string

**Comment by ParthivMangukiya** [comment]
@iamareebjamal i have added common login to continue string in this PR. Please Review.

File: android/app/src/main/java/org/fossasia/openevent/core/faqs/FAQFragment.java [hunk 1]
```diff
@@ -122,11 +125,13 @@ private void handleVisibility() {
...
-        super.onResume();
-        if (!AuthUtil.isUserLoggedIn()) {
-            tvEmptyFaqs.setText(R.string.login_to_view_faqs);
+    private void handleVisibilityUserLogin(boolean isUserLoggedIn){
+        if(isUserLoggedIn){
+            swipeRefreshLayout.setVisibility(View.VISIBLE);
+            loginBtn.setVisibility(View.GONE);
+        }else{
+            swipeRefreshLayout.setVisibility(View.GONE);
+            loginBtn.setVisibility(View.VISIBLE);
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/core/faqs/FAQFragment.java]: Syntax issues

File: android/app/src/main/java/org/fossasia/openevent/core/faqs/FAQFragment.java [hunk 2]
```diff
@@ -122,11 +125,13 @@ private void handleVisibility() {
         }
     }
 
-    @Override
-    public void onResume() {
-        super.onResume();
-        if (!AuthUtil.isUserLoggedIn()) {
-            tvEmptyFaqs.setText(R.string.login_to_view_faqs);
+    private void handleVisibilityUserLogin(boolean isUserLoggedIn){
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/core/faqs/FAQFragment.java]: Change name to showFaqLayout

Comment (ParthivMangukiya) [reply android/app/src/main/java/org/fossasia/openevent/core/faqs/FAQFragment.java]: done

**Review by iamareebjamal — APPROVED** [review]

**Comment by iamareebjamal** [comment]
TODO: Move logic to ViewModel

**Pull request merged**